examples/pxScene2d/src/pxWayland.cpp

```diff
@@ -569,45 +524,29 @@ rtError pxWayland::startRemoteObjectLocator()
 rtError pxWayland::connectToRemoteObject()
 {
   rtError errorCode = RT_FAIL;
-#ifdef ENABLE_PX_WAYLAND_RPC
-  int findTime = 0;
 
-  while (findTime < MAX_FIND_REMOTE_TIMEOUT_IN_MS && mClientPID != -1)
+#ifdef ENABLE_PX_WAYLAND_RPC
+  rtLogInfo("Attempting to find remote object %s", mRemoteObjectName.cString());
+  errorCode = rtRemoteLocateObject(mRemoteObjectName.cString(), mRemoteObject, 1000);
```

Comment:
In Wayland-Plugin, following Macros are used to keep on locating remote object for 5 seconds(5000 MS) at the interval of 100MS.
**_#define MAX_FIND_REMOTE_TIMEOUT_IN_MS 5000
#define FIND_REMOTE_ATTEMPT_TIMEOUT_IN_MS 100_**